run-cnf-suites.sh

```diff
@@ -41,12 +41,12 @@ while [[ $1 == -* ]]; do
 			fi ;;
 		-s|--skip)
 			while (( "$#" >= 2 )) && ! [[ $2 = --* ]] && ! [[ $2 = -* ]] ; do
-				SKIP="$2|$SKIP"
+				SKIP="(^$2)|$SKIP"
```

Comment:
Is this working for you? I could not get it to work on my machine. Looks like other regex are working but not ^. Also the tests seem to skip all

Reply:
Yeah I'm seeing the skip-all happening as well.

```
Ran 0 of 61 Specs in 0.002 seconds
SUCCESS! -- 0 Passed | 0 Failed | 0 Pending | 61 Skipped
--- PASS: TestTest (22.43s)
PASS
```

I'm just trying to do `make build-cnf-tests && ./run-cnf-suites.sh -f affiliated-certification`

Reply:
Yes, me too, I manually tested the regex beforehand and it "should" work, but in my case, if you see the DCI job below, it's also skipping all the tests. I'll take a look